lib/versionbot.ts

```diff
@@ -389,7 +389,7 @@ export class VersionBot extends GithubBot.GithubBot {
             // Now we get the new version from the CHANGELOG (*not* the package.json, it may not exist).
             return exec(`cat ${versionData.fullPath}${_.last(moddedFiles)}`).then((contents: string) => {
                 // Only interested in the first match for '## v...'
-                const match = contents.match(/^## (v[0-9]\.[0-9]\.[0-9]).+$/m);
+                const match = contents.match(/^## (v[0-9]+\.[0-9]+\.[0-9]+).+$/m);
```

Comment:
`versionist` currently doesn't have an option to return the calculated semantic version based on the commit history right?